packages/react-charting/src/components/StackedBarChart/MultiStackedBarChart.base.tsx

```diff
@@ -295,7 +295,9 @@ export class MultiStackedBarChartBase extends React.Component<IMultiStackedBarCh
         >
           <rect
             key={index}
-            id={`${this._barId}-${index}`}
+            id={`${this._barId + Math.floor(Math.random() * 100) + Math.floor(Math.random() * 10)}-${
```

Comment:
don't introduce random in the id. It is not good for react optimization and our tests.

Reply:
I figured out some issued
earlier we are using `this._barId = getId('_MSBC_rect_');` which create only 1 id eg `_MSBC_rect_3-${index}` for all 3 bars and its rect.
this._barId is same for all 3 bars and we are not using anywhere else. we can directly take the barIndex instead. `_MSBC_${barIndex}_${rectangle index}`